gapless5.js

```diff
@@ -1189,12 +1292,25 @@ function Gapless5(options = {}, deprecated = {}) { // eslint-disable-line no-unu
     this.uiDirty = true;
   };
 
+  /**
+   * @returns {string[]}
+   */
   this.getTracks = () => this.playlist.getTracks();
 
+  /**
+   * @returns {string[]} - audio path for current track, '' if none
```

Comment:
Isn't this supposed to be
```javascript
* @returns {string}
```
?

Reply:
Oops, thanks for catching that, fix committed!